packages/js/eslint-plugin/configs/recommended.js

```diff
@@ -58,15 +58,15 @@ module.exports = {
 				ignoreRestSiblings: true,
 			},
 		],
-		'react/react-in-jsx-scope': 'error',
+		'react/react-in-jsx-scope': 'off',
```

Comment:
Changed this rule since React 18 doesn't require React import for JSX

Reply:
Should we consider running the [codemod](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports) that removes the now-unnecessary react imports?

Reply:
Great suggestion @rjchow! I took care of that and opened the PR for review.